README.md

```diff
@@ -6,6 +6,14 @@
 
 The Go [OpenTelemetry](https://opentelemetry.io/) client.
 
+## Contributing
```

Comment:
I notice most other repos have a CONTRIBUTING.md file.
Consider creating such a file and moving this text there.

Reply:
@krnowak will you modify this PR to fix this or another PR?

Reply:
I have changed this PR.